docs/launcher.rst

```diff
@@ -0,0 +1,101 @@
+.. _Launcher:
+
+Launcher
+=========
+
+On this page we describe the different launchers for experiments. It is possible to use the local
+machine or batch schedulers like `Slurm <https://slurm.schedmd.com/overview.html>`_ or
+`Oracle Grid Engine <https://docs.oracle.com/cd/E19680-01/html/821-1541/docinfo.html#scrolltoc>`_
+(formerly SGE) to run experiments.
+
+In the following sections, we consider the
+`sorting expample <https://github.com/hu-macsy/simexpal/tree/master/examples/sorting>`_ from the
+:ref:`QuickStart` guide.
+
+..
+    TODO: Add section on ForkLauncher, Slurm, SGE
+
+Queue Launcher
+--------------
+
+As the name suggests, the Queue Launcher puts the jobs in a queue and works them off sequentially.
+It is possible to start a Queue Launcher as a daemon or interactively in a terminal. You can launch
+experiments and monitor/stop/kill the launcher.
+
+Starting the Launcher
+^^^^^^^^^^^^^^^^^^^^^
+
+Terminal
+~~~~~~~~
+
+You can start the Queue Launcher in a terminal so that it is possible to directly see internal
+states of the launcher. You will then work from another terminal window in order to send commands to
+the launcher. To start the launcher, use ``simex queue interactive``:
+
+.. code-block:: bash
+
+    $ simex queue interactive
+    Serving on /home/username/.extl.sock  # internal message of the Queue Launcher
```

Comment:
Move this paragraph below "As Daemon". I think this is not really that useful outside of debugging simexpal itself (it basically does the same as the fork launcher, I guess).